src/components/PlanningTool.jsx

```diff
@@ -387,19 +410,87 @@ class PlanningTool extends Component {
     if (this.actions.length <= 0) {
```

Comment:
This method is very long, would it be possible to refactor it? For instance, I would create new method after each `if`

Reply:
✅